Apps/fastlane/Fastfile

```diff
@@ -79,8 +64,11 @@ platform :ios do
       build_path: "build", # Save derived data to Apps/X/build folder. CI will parse this folder later for tracking SDK size. 
       cloned_source_packages_path: "spm_packages", # Save SPM dependencies to Apps/X/spm_packages folder. We can cache this folder to speed up builds on CI. 
     )
+    
+    are_environment_variables_set_for_build_uploading = ENV["FIREBASE_APP_DISTRIBUTION_SERVICE_ACCOUNT_CREDS_B64"]
 
-    are_environment_variables_set_for_build_uploading = !ENV["FIREBASE_APP_DISTRIBUTION_SERVICE_ACCOUNT_CREDS_B64"].empty?
+    environment_variables_required_for_build_uploading = ["FIREBASE_APP_DISTRIBUTION_SERVICE_ACCOUNT_CREDS_B64"] # array of keys 
+    are_environment_variables_set_for_build_uploading = environment_variables_required_for_build_uploading.all? { |key| ENV[key] != nil && !ENV[key].empty? } # check if all environment variables are set and not empty 
```

Comment:
I feel like this can be simplified 

```
# Constants should be defined using uppercase letters and underscores
REQUIRED_ENV_VARS = ["FIREBASE_APP_DISTRIBUTION_SERVICE_ACCOUNT_CREDS_B64"]

# Check if all required environment variables are set and not empty
are_environment_variables_set_for_build_uploading = REQUIRED_ENV_VARS.all? { |key| ENV[key].to_s.strip.length > 0 }
```

Reply:
I do appreciate the suggestion. I am going to suggest we keep what the PR has because this line of code has caused issues in the past and this code seems to be stable from my testing. Not being ruby experts, I prefer using code that works over code that can be simplified.